src/style.css

```diff
@@ -1,3 +1,72 @@
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  height: 40px;
+  width: 40px;
+}
+
+.nav {
+  display: block;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav__items {
+  margin-left: 20px;
+}
+
+.nav__items:first-child {
+  margin-left: 0;
+}
+
+.nav__link:link {
+  position: relative;
+  display: flex;
+  text-decoration: none;
+  color: #000;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  height: 60px;
+  align-items: center;
+  cursor: pointer;
+  transition: color 0.3s;
+}
+
+.nav__link:visited {
+  color: #000;
+}
+
+.nav__link.is-active {
```

Comment:
```suggestion
.is-active {
```

Reply:
i don't know why, but it doesn't pass the test with this changings. only when i change selector on: a.nav__link.is-active...